paper-slider.html

```diff
@@ -292,7 +292,7 @@
      * @method increment
      */
     increment: function() {
-      this.value = this._clampValue(this.value + this.step);
+      this.value = this._clampValue(this.value + this.step).toString();
```

Comment:
Sigh... see https://github.com/PolymerElements/paper-slider/issues/22. Basically IronFormElementBehavior makes it a string.